content/pt/blog/news/how-to-build-one-effective-middleware.md

```diff
@@ -84,15 +84,15 @@ func mid2() gin.HandlerFunc {
 }
 ```
 
-According to [Constituent parts](#Constituent-parts) said, when we run the gin process, **part one** will execute firstly and will print the following information:
+De acordo com as [partes constituintes](#partes-constituintes) disseram, quando executamos o processo de `gin`, a **parte um** executará em primeiro lugar e imprimirá a seguinte informação:
```

Comment:
Portuguese has a different way of expressing ideas, so if you put your translated text in the same order as English (using the same punctuation marks in the wrong places), you will break the main idea of the documentation. Please have a look at this.

Beside, you did a great job, congrats @nazarepiedady 🎉

Reply:
@overlineink This time you got me, some translations I did in a hurry.